Tribes/src/app/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Notification } from './../notification';
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-progress-bar',
+  templateUrl: './progress-bar.component.html',
+  styleUrls: ['./progress-bar.component.css']
+})
+
+export class ProgressBarComponent implements OnInit {
+
+  @Input() notification: Notification;
+  result: number;
+
+  constructor() { }
+
+  ngOnInit() {
+    console.log(this.checkTimeDifference(this.notification));
+  }
+
+  checkTimeDifference(notification: Notification): number {
+    const currentTime = new Date().getTime();
+    const timeDiff = (notification.finishedAt - currentTime) / 1000 / 60;
```

Comment:
If you are storing minutes you could call it that :)

Reply:
renamed variable to minuteDiff and totalMinutes